cmd/kube-apiserver/app/server.go

```diff
@@ -315,15 +315,19 @@ func Run(s *options.ServerRunOptions) error {
 		sets.NewString("watch", "proxy"),
 		sets.NewString("attach", "exec", "proxy", "log", "portforward"),
 	)
+	genericConfig.RESTOptionsGetter = &kubeapiserver.RESTOptionsFactory{
```

Comment:
A comment wouldn't harm, that this is overwriting the simple rest options factory due to special kube requirement. People should not copy this pattern, a comment mgiht help.